src/scripts/lib/bugsnag.ts

```diff
@@ -16,6 +16,7 @@ function processError(err: Error) {
       /.*(moz-extension|chrome_extension|chrome-extension|file):\/\/.*\/scripts\/(.*)/gi,
       'togglbutton://scripts/$2'
     );
+    frame.file = frame.file.replace(/\\/g, '/');
```

Comment:
Today we know what it is about. Tomorrow we may not. Can we leave some comment? :pray: